pxtcompiler/emitter/emitter.ts

```diff
@@ -2134,6 +2134,9 @@ ${lbl}: .short 0xffff
                 return emitExpr(args[0])
             }
 
+            if (opts.target.shimRenames && U.lookup(opts.target.shimRenames, nm))
+                nm = opts.target.shimRenames[nm]
```

Comment:
nit, all in 1 shot?

    nm = opts.target?.shimRenames[nm] || nm

Reply:
I think that would protect against `opts.target` being `null`, not `opts.target.shimRenames`

Reply:
I guess there's `opts.target.shimRenames.?[nm]`...

Reply:
or something